source/docs/software/advanced-controls/introduction/approaches-to-ctrl-sys-design.rst

```diff
@@ -0,0 +1,77 @@
+
+When desigining a control system for your robot, there are a number of different approaches to take. They range from very simple approaches, to advanced and complex ones. Each has *tradeoffs*. Some will work better than others in different situations, some require more mathematical analysis than others.
+
+Teams should prioritize picking the easiest strategy which enables success on the field. However, as you do experiemnts, keep in mind there is almost always a "next-step" to take to improve your field performance.
+
+FeedForward vs. FeedBack
+------------------------
+
+There are two broad categories of strategies in control system design:
+
+**Feed-Forward** refers to the class of algorithms which incorporate knowledge of how the mechanism under control is *expected* to operate. Using this "model" of operation, the control input is chosen to make the mechanism get close to where it should be.
+
+**Feed-Back** refers to the class of algorithms which use sensors to *measure* what a mechanism is doing, and issue corrective commands to move a mechnaism from where it actually is, to where you want it to be.
+
+They are not mutually exclusive - many techniques will use these concepts simultaneously.
+
+Start with FeedForward
+-----------------------
+
+Because most FRC mechanisms can be manufactured to be have well-defined behavior and can be easily modeled, it's generally encouraged to look into feed-forward techniques first. 
+
+How do you expect your system to behave?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step in designing a control system is writing down information on how you expect your mechanism to behave.
+
+This step is done by combining one or more concepts you may be familiar with from Physics: drawing free body diagrams of the forces that act on the mechanism, taking measurements of mass and moment of inertia from your CAD models, applying standard models of how DC motors or pneumatic cylinders convert energy into mechanical force and motion, etc.
+
+Models do not have to be extremely accurate to be useful. For example, a very simple model of a motor is that its speed is proportional to the voltage applied. A motor which spins at 3000 RPM at 12v will spin near 1500RPM at 6V. If you know you want your motor to be spinning at 1254 RPM, you can use the voltage/speed ratio to calculate the amount of voltage to send to the motor.
+
+`ReCalc is an online calculator <https://www.reca.lc/>`__ which helps take care of some of these physics model calcualtions for common FRC mechanisms.
+
+
+System Identification
+^^^^^^^^^^^^^^^^^^^^^
+
+A key way to improve the accuracy of a simple physics model is to perform experiments on the real mechanism, record data, and use the data to *derive* the constants associated with different parts of the model.
+
+This is useful if CAD models are not perfectly accurate, or parts of the model are not easily predicted (ex: friction in a gearbox).

```

Comment:
I'd combine this sentence with the previous paragraph, and be specific with reference to equations if possible (if the feedforward equations are introduced in the previous section, you can reference the simple motor equation with and without friction here, and talk about how it's nearly impossible to *predict* `kS` from system parameters